pandas/tests/window/test_pairwise.py

```diff
@@ -296,7 +298,12 @@ def test_no_pairwise_with_self(self, pairwise_frames, pairwise_target_frame, f):
             lambda x, y: x.expanding().cov(y, pairwise=True),
             lambda x, y: x.expanding().corr(y, pairwise=True),
             lambda x, y: x.rolling(window=3).cov(y, pairwise=True),
-            lambda x, y: x.rolling(window=3).corr(y, pairwise=True),
+            # TODO: We're missing a flag somewhere in meson
```

Comment:
I bet meson is actually more correct but we probably have a longstanding bug. I think fine to skip for now, but separately and related to comment above want to figure out the best way to configure compiler warnings and see what is being passed. I would be very surprised if there wasn't an error highlighting the bug in our code here

Reply:
Yeah if you could link to a build where this failed I'd be curious to see what the precision loss is.

Reply:
https://github.com/pandas-dev/pandas/actions/runs/3497030648/jobs/5855635374.

It's the last number.

Reply:
> I bet meson is actually more correct but we probably have a longstanding bug.

https://github.com/pandas-dev/pandas/issues/39553?

> It's the last digit of the last number in the array.

https://github.com/pandas-dev/pandas/issues/39553#issuecomment-773794670?